docs/trex_getstarted.md

```diff
@@ -94,30 +91,80 @@ To use the dashboard extension samples, you need to start up a web server on you
+
+    ```javascript
+    (function () {
+      $(document).ready(function () {
+        tableau.extensions.initializeAsync().then(function () {
+        /* body of function  */
+        /* controls what the extension does */
+        /* extension calls other functions here */ 
+        } function (err) {
+        // Something went wrong in initialization.
+           console.log('Error while Initializing: ' + err.toString());
+        });
+      });
+    /*  extension can define other functions here as needed */
+    })();
+    
+    ```
+  
+4. Open the `DataSources.trex` file. This is the manifest file for the extension. This is the file that you selected to add the extension to the dashboard. This file defines certain properties for the extension, such as the name, and author, and the location (URL) of the extension. 
+ 
+    ```xml
+    <source-location>
+      <url>http://localhost:8765/Samples/DataSources/datasources.html</url>
+    </source-loc>
+    ```
+    If you make a copy of the sample directory so that you can start to modify the code and experiment with the Extensions API, you just need to modify this path so that the URL reflects the new location.
+
+    ```xml
+    <source-location>
+      <url>http://localhost:8765/_your-new-folder-here_/DataSources/datasources.html</url>
+    </source-loc>
+    ```
+
+
+<!-- ### Using dashboard extensions in a workbook
 When a Tableau Extension is installed, you can use the extension like you would any other dashboard object. The settings for the extensions are saved when you save the workbook. 
 You can add multiple instances of an extension to a dashboard or to multiple dashboards within a workbook. The settings for each instance are saved separately.
-
+ 
+ -->
 
 
 ------------------------------------------------------------------------
   
 ## What's next?
 
-- For more information about how you can use Extensions API, go look at the [Samples (GitHub)](https://github.com/tableau/extensions-api/tree/master/Samples/), or follow the [Tutorial (GitHub)](https://github.com/tableau/extensions-api/tree/master/Tutorial) and learn how to build a dashboard extension, step by step. If you downloaded or cloned the repository, look in the `Samples` and `Tutorial` folder on your computer.  
+* Start developing your extension by modifying an existing sample. See [samples (GitHub)](https://github.com/tableau/extensions-api/tree/master/Samples/). Create a copy of the sample directories. For example, if you make the copy in `\extensions-api\` directory, and call it `MySamples`, you just need to modify the URL in the `.trex` files so that you host the extensions using the same web server you created with `npm start` command.
```

Comment:
Good point. The link is just there for convenience. They can stay in the browser and see the sample directories on GitHub. I'll add text about "if you downloaded or ..." because that's what we want them to do.